tests/data/checkpoint/checkpoint_for_testing/model_config.yaml

```diff
@@ -0,0 +1,144 @@
+
+wandb:
+  project_name: modalities
+  mode: ONLINE
+
+data:
+  sample_key: "input_ids"
+  target_key: "target_ids"
+  sequence_len: 128
+  train_dataloader:
+    type_hint: LLMDataLoader
+    config:
+      dataloader_tag: "train"
+      num_workers: 2
+      pin_memory: true
+      shuffle: false
+      batch_sampler:
+        type_hint: BatchSampler
+        config:
+          batch_size: 8 # per rank
+          drop_last: false
+          sampler:
+            type_hint: DistributedSampler
+            config:
+              rank: ${training.global_rank}
+              num_replicas: ${training.world_size}
+              shuffle: true
+      dataset:
+        type_hint: PackedMemMapDatasetContinuous
+        config: 
+          raw_data_path: /home/richard-rutmann/s3/projects/remote_projects/modalities/data/sample_datasets/redpajama_v2/mem_map/redpajama_v2_samples_1024_train.pbin
+          block_size: ${data.sequence_len}
+          sample_key: ${data.sample_key}
+      collate_fn:
+        type_hint: GPT2LLMCollator
+        config:
+          sample_key: ${data.sample_key}
+          target_key: ${data.target_key}
+  eval_dataloaders:
+    - type_hint: LLMDataLoader
+      config:
+        dataloader_tag: "val"
+        num_workers: 2
+        pin_memory: true
+        shuffle: false
+        batch_sampler:
+          type_hint: BatchSampler
+          config: 
+            batch_size: 8 # per rank
+            drop_last: false
+            sampler:
+              type_hint: DistributedSampler
+              config:
+                rank: ${training.global_rank}
+                num_replicas: ${training.world_size}
+                shuffle: false
+        dataset:
+          type_hint: PackedMemMapDatasetContinuous
+          config: 
+            raw_data_path: /home/richard-rutmann/s3/projects/remote_projects/modalities/data/sample_datasets/redpajama_v2/mem_map/redpajama_v2_samples_1024_test.pbin
+            block_size: ${data.sequence_len}
+            sample_key: ${data.sample_key}
+        collate_fn:
+          type_hint: GPT2LLMCollator
+          config: 
+            sample_key: ${data.sample_key}
+            target_key: ${data.target_key}
+
+training:
+  process_group_backend: "nccl"
+  global_num_training_samples: 2048
+  callback_interval_in_samples: 12000
+  local_rank: ${oc.env:LOCAL_RANK}
+  global_rank: ${oc.env:RANK}
+  world_size: ${oc.env:WORLD_SIZE}
+  main_rank: 0
+  local_train_micro_batch_size: ${data.train_dataloader.config.batch_sampler.config.batch_size}
+  global_num_seen_samples: ${modalities_setup.settings.global_num_seen_samples}
+  gradient_acc_step: 1
+  do_apply_activation_checkpointing: false
+
+checkpointing:
+  checkpointing_strategy:
+    type_hint: SaveKMostRecentCheckpointsStrategy
+    config:
+      k: -1   # -1 to save all checkpoints
+  checkpointing_execution:
+    type_hint: FSDPToDiscCheckpointing
+    config: 
+      checkpoint_path: /home/richard-rutmann/s3/models/llm_gym/test/checkpoints
```

Comment:
@rrutmann here as well

Reply:
Done in 595e007bea194fb4cad04b09502464c50f79222a